ddl/ddl_api.go

```diff
@@ -343,7 +343,7 @@ func (d *ddl) ModifySchemaSetTiFlashReplica(sctx sessionctx.Context, stmt *ast.A
 		return infoschema.ErrDatabaseNotExists.GenWithStackByArgs(dbName.O)
 	}
 	if util.IsMemOrSysDB(dbInfo.Name.L) {
-		return errors.Trace(dbterror.ErrUnsupportedAlterReplicaForSysTable)
+		return errors.Trace(dbterror.ErrUnsupportedTiFlashOperationForSysTable)
 	}
```

Comment:
Seems that this check duplicates with checkTiFlashSupportable?

Reply:
It only check the MemOrSysTable here, I think we can keep it as it is.